active/google-summer-of-code.md

```diff
@@ -0,0 +1,61 @@
+# Google Summer of Code Working Group
+
+## Scope of responsibilities
```

Comment:
One extra point that I would like to see here:

* Actively work with other Django WG's to identify improvements for GSoC potential participants in the community and for the community to help potential participants.

The above could a number of forms, but essentially I see it about sign-posting information in a more targeted manner and identifying those potential participants to push them along in the right direction. This could be an extra page on the site with a form or specific channels in the Forum/Discord for GSoC discussion.

Reply:
I think this is a 'must' of working with other groups, I kept it vague and high level to allow that in my head, how that happens is a could thing to be discussed and decided.

I think of my suggestion as more of a collaboration at this point. Given the amount of 'spam' (I know it's not actual spam) that crops up in both the Forum/Discord I'm thinking of implementing some ideas as part of the OCWG, but it would better to do this in collaboration with this WG in a coordinated manner rather than independently.

Reply:
Great, thank you! Added to the PR. Marking this as resolved.